oceannavigator/frontend/src/components/DatasetSelector.jsx

```diff
@@ -134,6 +136,14 @@ export default class DatasetSelector extends React.Component {
       ];  
     }
 
+    let quiverVariables = [];
+    if (this.props.datasetVariables) {
+      quiverVariables = this.props.datasetVariables.filter((variable) => {
+        return variable.id.includes("mag") && variable.id.includes("vel");
+      })
+    }
+    quiverVariables.unshift({ id: 'none', value: 'None' });
```

Comment:
Add  `none` to the beginning of the options list so it's the default option selected when the page loads.